app/models/manageiq/providers/openshift/container_manager/container_image.rb

```diff
@@ -1,10 +1,48 @@
 class ManageIQ::Providers::Openshift::ContainerManager::ContainerImage < ContainerImage
-  def annotate_deny_execution(causing_policy)
+  def annotate_image(annotations)
     ext_management_system.annotate(
       "image",
       digest,
+      annotations
+    )
+  end
+
+  def openscap_summary
+    failed_rules = openscap_rule_results.where(:result => "fail")
+    [[['High'], 'Critical', 3],
+     [['Medium'], 'Important', 2],
+     [['Low'], 'Medium', 1],
+     [['Info', 'Unknown'], 'Low', 0]].collect do |severities, label, index|
+      {
+        :label         => label,
+        :severityIndex => index,
+        :score         => failed_rules.select { |rule| severities.include?(rule.severity) }.count
```

Comment:
I think this is called `data` in https://github.com/adellape/openshift-docs/blob/master/security/container_content.adoc#container-content-scanning no?